http_cache/validation.rst

```diff
@@ -236,3 +236,6 @@ headers that must not be present for ``304`` responses (see
 :method:`Symfony\\Component\\HttpFoundation\\Response::setNotModified`).
 
 .. _`expiration model`: https://tools.ietf.org/html/rfc2616#section-13.2
+.. _`HTTP ETag`: https://en.wikipedia.org/wiki/HTTP_ETag
+.. _`DeflateAlterETag`: https://httpd.apache.org/docs/trunk/mod/mod_deflate.html
+.. _`BrotliAlterETag`: https://httpd.apache.org/docs/2.4/mod/mod_brotli.html
```

Comment:
You said starting from 2.5 you can control this, but this link is to 2.4 and the above is to `trunk` 🤔

Reply:
Good catch! I removed the reference to Apache 2.5 because one directive is for 2.4+ and the other one for 2.5+. So, let's just link to Apache docs and readers will find all the information they need. Thanks.